exercise-2/style.css

```diff
@@ -0,0 +1,78 @@
+.first-container{
+  width: 348px;
+  padding: 0px 0px 0px 30px;
+  margin: 40px 0px 0px 34px;
+  text-align: center;
+  font-family: 'Raleway';
+}
+.second-container{
+  width: 348px;
+  margin: -233px 0px 0px 438px;
```

Comment:
what is the use of this negative margin? Pls find a better way of alignment.

Reply:
Can we use position :relative in this case ?